packages/nx/src/hasher/hasher.ts

```diff
@@ -84,10 +84,6 @@ export class Hasher {
     const legacyFilesetInputs = [
       ...Object.keys(this.nxJson.implicitDependencies ?? {}),
       'nx.json',
-      //TODO: vsavkin move the special cases into explicit ts support
```

Comment:
It looks like are handling it already in hashExternalDependency but the comment there is incorrect. Can you fix it?